MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -6887,15 +6854,15 @@ public PartInventory getPartInventory(Part part) {
             }
             if (part.isSamePartType(p)) {
                 if (p.isPresent()) {
-                    if (p instanceof Armor) { // ProtomekArmor and BaArmor are derived from Armor
```

Comment:
Revert these `ProtomekArmor` -> `ProtoMech Armor` changes. We're referring to class names here.